.travis.yml

```diff
@@ -98,6 +98,8 @@ install:
 script: |
   set -e
   if [[ $JOB = docs ]]; then
+    # install docs dependencies
+    pip install '.[docs]'
```

Comment:
I think you can. I assume it is more versatile like this. If we tell travis to `pip install mock` we have to change it if we ever change our requirements. If we tell travis to `pip install .[docs]` pip will look at our setup.py file and check the extra_requires stuff for a "docs" key (I assume). In this way we only have to change the setup.py if we change our requirements and it will this be already communicated to travis and any other (end) user using pip/setup.py (so hopefully everybody).